lingua_franca/format.py

```diff
@@ -370,6 +374,39 @@ def nice_date_time(dt, lang='', now=None, use_24hour=False,
                                              use_ampm)
 
 
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def nice_day(dt, date_format='MDY', include_month=True, lang=""):
+    if include_month:
+        month = nice_month(dt, date_format, lang)
+        if date_format == 'MDY':
+            return "{} {}".format(month, dt.strftime("%d"))
+        else:
+            return "{} {}".format(dt.strftime("%d"), month)
+    return dt.strftime("%d")
```

Comment:
just realized that a one digit day is formatted with "0x". I don't know if all TTS can handle that.
Better `dt.strftime("%-d")`